app/src/main/res/layout/search_widget_extra_small_v1.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<!-- FrameLayout is needed to properly align the widget in the home screen. -->
```

Comment:
The comment isn't actually true it seems. Feel free to ask me questions later, but I can show that it looks fine without the outer layout.